python/overrides.js

```diff
@@ -22,4 +22,5 @@ module.exports = {
     // TODO verify and enable
     'appium iOS check fully window with scroll and pageCoverage': { skipEmit: true },
     'appium iOS check window region with scroll and pageCoverage': { skipEmit: true },
+    'appium iOS nav bar check regio': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9214167)
```suggestion
    "appium iOS nav bar check regio": {skipEmit: true},
```